public_suffix_list.dat

```diff
@@ -10857,6 +10857,11 @@ bnr.la
 // Submitted by Paul Crowder <paul.crowder@blackbaud.com>
 blackbaudcdn.net
 
+// BoltN Hosting Limited t/a BoltN Group : https://boltn.group/
+// Submitted by Matthias Merkel <matthias@boltn.group>
+pages.gitcloud.app
+*.pages.gitcloud.app
```

Comment:
why with the "*" ??

Reply:
There are instances where a fourth level domain belongs to a different user than a third level domain

Reply:
I have removed the *.pages.gitcloud.app one

Reply:
@Moritz30 From your description, it sounds like it might have been the correct one. Can you provide a list of sample domains to help make sure we've got the one most appropriate for your needs?

Reply:
@sleevi Some examples of domains that could be owned by different users:

foo.pages.gitcloud.app
bar.pages.gitcloud.app
bar.foo.pages.gitcloud.app
foo.bar.pages.gitcloud.app